### DISCUSSION TITLE
Placeholder text for Option Button

### DISCUSSION THREAD

**Discussion by Koomball** [godotengine/godot-proposals#10543] [GUI]
The option button is missing the option to add placeholder text, i feel this doesn't make sense as all the other button types have this option.

**Comment by AThousandShips** [comment]
What placeholder text are you talking about? No button has this that I can tell

**Reply by Koomball** [reply]
<img width="960" alt="Screenshot (367)" src="https://github.com/user-attachments/assets/37a563c5-a693-43fa-be6b-9759893c212d">
<img width="960" alt="Screenshot (368)" src="https://github.com/user-attachments/assets/0674ac54-05d5-4563-a179-745ea50072c3">

**Reply by AThousandShips** [reply]
The text will be displayed if you select one of the options, but there's no "option" in the other buttons, just that they default to having some text (do you mean that the new items in have their text be empty instead of having some placeholder text?)

**Reply by Koomball** [reply]
I mean that each of the other buttons have an option to add text that can be used to describe what the button does, which is something the option button doesn't have.

**Reply by AThousandShips** [reply]
Oh that's very different, my bad you mean just text, there's nothing placeholder about a the text of a button that's why I was confused

**Reply by Koomball** [reply]
sorry should of just said text aha, i was saying placeholder due to when you select an item it would replace the text with the selected option.

**Reply by AThousandShips** [reply]
For that you can add an option that's what ever default text you want, and then have it be selected by default, that's how a lot of interfaces do it (for example, on a web page where it says "Select State" and it lists states *and* the "Select State" option)

**Comment by Christopher-Temple** [comment]
Whatever text you want to be the default "placeholder" text, put that as your top option. Then under your Inspector, used the Selected option to pick that option. Then, make sure you toggle Separator on that item as well. 

This makes your selected option the default text, and when you click on the button to drop down the options, the top option will be surrounded by a separator, like this....

-----Choose A Number-----
o 1
o 2
o 3
o 4

If you are running any kind of checks to make sure an option is chosen, the default selected does not count since it's labeled as a separator.
